packages/apollo/README.md

```diff
@@ -143,7 +143,8 @@ OPTIONS
 
   --key=key                                  The API key for the Apollo Engine service
 
-  --localSchemaFile=localSchemaFile          Path to your local GraphQL schema file (introspection result or SDL)
+  --localSchemaFile=localSchemaFile          Path to your local GraphQL schema file (introspection result or SDL as 
```

Comment:
changing the description of a flag actually doesn't happen here! This portion of the file is generated by oclif. You'd need to change this in `packages/apollo/src/commands/client/codegen.ts` along with changing any other uses of the `localSchemaFile` flag, like in the service commands as well.